src/guessTheNumber.js

```diff
@@ -0,0 +1,18 @@
+const min = Math.ceil(1)
+const max = Math.floor(100)
+const randomNum = Math.floor(Math.random() * (max - min + 1) + min)
+const getUserInput = document.querySelector('input#guess-number')
+const guessTheNumber = (inputValue) => {
+  inputValue = getUserInput.value
+  if(inputValue.toLowerCase() === 'exit') {
+    alert('Game Over! Reload browser to restart game!')
```

Comment:
Once you add tests, you won't be able to test alerts. You might want to switch to returning a string.